draft-ietf-jsonpath-base.md

```diff
@@ -1102,11 +1102,10 @@ enclosing the identifier).
 A test expression either tests the existence of a node
 designated by an embedded query (see {{extest}}) or tests the
 result of a function expression (see {{fnex}}).
-In the latter case, if the function expression is of type
-`ST(OptionalBooleanType)` (see {{typesys}}), it tests whether the result
-is `true`; if the function expression is of type
-`ST(OptionalNodesType)`, it tests whether the result is different from
-`Nothing`.
+In the latter case, the function expression MUST be of type
+`BooleanType` (see {{typesys}}) or of type `NodelistType` to appear
+in a test expression.  The expression tests whether the result
+is or can be converted to `PropositionTrue` (see {{type-conv}}).
```

Comment:
`PropositionTrue` should be `BooleanTrue`.

Reply:
Thanks.  I had changed to "proposition" as in another PR, but then decided against it.  Must've missed this one coming back.